core/server/data/importer/index.js

```diff
@@ -293,9 +294,7 @@ _.extend(ImportManager.prototype, {
     loadFile: function (file) {
         var self = this,
             ext = path.extname(file.name).toLowerCase();
-
-        this.filesToDelete.push(file.path);
```

Comment:
Why was this changed to an empty array and not something like `self.filesToDelete = [file.path];` ? :thinking: